library/tasks/Picard.wdl

```diff
@@ -95,15 +103,16 @@ task CollectDuplicationMetrics {
        METRICS_FILE="${output_filename}.duplicate_metrics.txt" \
        REMOVE_DUPLICATES=false
   }
+
   runtime {
```

Comment:
A thought on ordering: Does it matter what order the components of the task are coded in? 

If not, the most coherent order, to me, would be: 
`runtime`, `command`, `output`.

Reply:
ordering doesn't matter at all.  I would argue that the `command` block is what people want to see first although if we're now putting in `meta` and `parameter_meta` blocks maybe those push other blocks lower?

Reply:
I guess I'm thinking about this in terms of dependencies. You put inputs first because it tells you what data and information you have access to. In the same way, I'd argue that runtime comes next because you want to know how much ram/cpu/disk resource you're able to consume before you see the actual command that gets run. 

Then, once you've run something, output comes last.